src/components/Card/Card.tsx

```diff
@@ -0,0 +1,16 @@
+import './Card.scss';
+import { Movie } from '../../types';
+
+interface CardProps {
+  movie: Movie;
+}
+
+export default function Card({ movie }: CardProps) {
```

Comment:
You could use a deep destructure here to get rid of the movie. syntax.

For example: { movie: { id, title, year } }